src/components/Health/HealthIndicator.tsx

```diff
@@ -50,43 +43,62 @@ export class HealthIndicator extends React.PureComponent<Props, HealthState> {
   }
 
   renderSmall(health: H.Health) {
-    return <span>{this.renderIndicator(health, '18px', '12px', this.state.globalStatus.name)}</span>;
+    const icon = this.renderIcon('18px', '12px');
+    return this.renderPopover(health, icon);
   }
 
   renderLarge(health: H.Health) {
+    const spanStyle: React.CSSProperties = {
+      color: this.state.globalStatus.color,
+      fontWeight: 'bold',
+      position: 'relative',
+      top: -9,
+      left: 10
+    };
     return (
-      <div style={{ color: this.state.globalStatus.color }}>
-        {this.renderIndicator(health, '35px', '24px', this.state.globalStatus.name)}
+      <div>
+        {this.renderIcon('35px', '24px')}
+        <span style={spanStyle}>{this.state.globalStatus.name}</span>
```

Comment:
Replace this div by <> </> please.

This is by https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html from version 16.2.0  . This avoid us to add extra div to the DOM

Reply:
Maybe we should add this to the Guides What do you think @jotak , @lucasponce, @xeviknal  ?

Reply:
done.